targets/TARGET_STM/us_ticker_16b.c

```diff
@@ -52,22 +50,32 @@ void us_ticker_init(void)
 
 uint32_t us_ticker_read()
 {
-    uint32_t counter;
-
     TimMasterHandle.Instance = TIM_MST;
```

Comment:
This TimMasterHandle.Instance initialization can be removed from here, because it will either have been already done previously, or it will be done in the us_ticker_init() call immediately below.